environment/mainnet.json

```diff
@@ -2,7 +2,23 @@
 	"environment_type": "production",
 	"matrix_server_list": "https://raw.githubusercontent.com/raiden-network/raiden-service-bundle/master/known_servers.main.test.yaml",
 	"pfs_with_fee": "https://pfs.rsb01.test.raiden.network",
-	"eth_rpc_endpoint": "http://geth.mainnet.ethnodes.brainbot.com:8545",
+        "eth_rpc_endpoints": [
```

Comment:
indention looks strange here

Reply:
The usual tab vs spaces problem. Do we always want to use spaces in JSON? Then I'll change my config so that I'll get that right in future commits.

Reply:
No strong feelings from my side

Reply:
Personally I think tabs should be eradicated from time ;)